builtin/rebase.c

```diff
@@ -1066,25 +1088,26 @@ int cmd_rebase(int argc, const char **argv, const char *prefix)
 			strbuf_reset(&buf);
 			if (capture_command(&stash, &buf, GIT_MAX_HEXSZ))
 				die(_("Cannot autostash"));
+			trim_trailing_newline(&buf);
```

Comment:
This should not be here, as it has nothing to do with the *config setting* but everything with `--autostash`. It should go into *that* commit.